devstudio/installer/category.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site>
+	<description>
+		JBoss Developer Studio Integration Stack Installer
+	</description>
+
+	<!-- copy IU's from JBDS Core Installer -->
+	<feature id="org.jboss.tools.bpel.feature" />
+	<feature id="org.testng.eclipse" />
+	<bundle id="com.jboss.devstudio.core.package" />
```

Comment:
hmm - this would result in the content generated for the extra updatesite embedded in the installer would be duplicate of what is in the main updatesite included would it not ?